docs/pages/tabs.js

```diff
@@ -96,6 +96,248 @@ card(
+        defaultCode={`
+        function OrderExample() {
+          const [activeIndex, setActiveIndex] = React.useState(0);
+
+          const handleChange = ({ activeTabIndex, event }) => {
+            event.preventDefault();
+            setActiveIndex(activeTabIndex)
+          };
+
+          const tabs = [
+            { href: "https://pinterest.com", text: "All" },
+            { href: "https://pinterest.com", text: "Type treatments" },
+            { href: "https://pinterest.com", text: "Font pairings" },
+          ];
+
+          return (
+            <Tabs
+              activeTabIndex={activeIndex}
+              onChange={handleChange}
+              tabs={tabs}
+            />
+          );
+        }
+          `}
+      />
+      <MainSection.Card
+        cardSize="md"
+        type="don't"
+        description="Disable or hide Tabs if a Tab's content is empty. There should always be at least 2 Tabs. We don't support applying a disabled state for the Tab as it can cause usability and accessibility issues."
+        defaultCode={`
+        function FilterExample() {
+          const [activeIndex, setActiveIndex] = React.useState(0);
+
+          const handleChange = ({ activeTabIndex, event }) => {
+            event.preventDefault();
+            setActiveIndex(activeTabIndex)
+          };
+
+          const tabs = [
+            { href: "https://pinterest.com", text: "Saved" },
+          ];
+
+          return (
+            <Tabs
+              activeTabIndex={activeIndex}
+              onChange={handleChange}
+              tabs={tabs}
+            />
+          );
+        }
+          `}
+      />
+    </MainSection.Subsection>
+  </MainSection>,
+);
+
+card(
+  <MainSection
+    name="Accessibility"
+    description="Tabs are intended for page-level navigation between multiple URLs.
+  Each tab must have an individual title that precisely describes the tab content. Provide a short, descriptive label for screen-readers using `accessibilityLabel`. It is helpful for users of assistive technologies so they have the necessary information to navigate the content efficiently."
+  >
+    <MainSection.Subsection
+      title="Keyboard"
+      description={`
+      Tab key navigates the tabs.
+      Enter/return key activates a tab (i.e., it navigates to the link href).`}
+      columns={2}
+    />
+    <MainSection.Subsection
+      title="Screen Reader"
+      description={`
+      The tab/link **must** announce a state of "current" if the HREF matches the current window URL.
+      The tab/link **must** announce "tab" behavior.`}
+    />
+    <MainSection.Card />
+  </MainSection>,
+);
+
+card(
+  <MainSection
+    name="Localization"
+    description={`Be sure to localize text and accessibilityLabel.
+    The Tab's title should be 3 words or less, long enough to be understood by users but short enough to prevent text wrapping. Aim for a single word when possible.`}
```

Comment:
nit: technically this is a run-on sentence with the comma. Ideally that should be a colon, since it's clarifying the previous clause